app-text/texlive-core/texlive-core-2023-r9.ebuild

```diff
@@ -459,7 +461,8 @@ src_install() {
 
 	dodir "/usr/bin"
 	for i in ${TEXLIVE_MODULE_BINLINKS} ; do
-		[[ -f ${ED}/usr/bin/${i%:*} ]] || die "Trying to install an invalid BINLINK ${i%:*}. This should not happen. Please file a bug."
+		[[ -f ${ED}/usr/bin/${i%:*} ]] \
```

Comment:
Sorry, the syntax I had in mind to avoid the redundant backslash is/was

```bash
		[[ -f ${ED}/usr/bin/${i%:*} ]] ||
			die "Trying to install an invalid BINLINK ${i%:*}. This should not happen. Please file a bug."
```

Don't feel urged to fix this. I'll do so when merging this.